src/commands/synthetics/__tests__/reporters/junit.test.ts

```diff
@@ -0,0 +1,126 @@
+// tslint:disable: no-string-literal
+import {promises as fs} from 'fs'
+import {Writable} from 'stream'
+
+import {getDefaultStats, JUnitReporter} from '../../reporters/junit'
+import {RunTestCommand} from '../../run-test'
+import {getApiTest, getResult, getStep} from '../fixtures'
+
+const globalTestMock = getApiTest('123')
+const globalStepMock = getStep()
+const globalResultMock = getResult()
+
+describe('Junit reporter', () => {
+  const writeMock: Writable['write'] = jest.fn()
+  const commandMock: unknown = {
+    context: {
+      stdout: {
+        write: writeMock,
+      },
+    },
+    jUnitReport: 'junit',
+  }
+
+  let reporter: JUnitReporter
+
+  describe('constructor', () => {
+    beforeEach(() => {
+      reporter = new JUnitReporter(commandMock as RunTestCommand)
+    })
+    it('should update destination with .xml if needed', () => {
```

Comment:
I don't understand the name of this test: when is the destination updated?

Reply:
It's just when we use `--jUnitReport=jUnit-report` for instance (without a `.xml`), it will append the `.xml` part.

Reply:
oh that makes sense and it's good that it does it. Maybe a clearer description would be 
`should append '.xml' to destination if isn't there` ?